src/services/file.services.js

```diff
@@ -167,7 +167,7 @@ export const deleteFileByPath = async (
     });
     console.log(`File '${path}' deleted successfully.`);
   } catch (error) {
-    if(error.status === 404) {
+    if(error.status === 404 || error.status === 422) {
```

Comment:
Why the 422 code is here? A 422 code means `Unprocessable Content`, which is related to when you send a request that has data that cannot be processed accordingly (for example, you are waiting for a body with string values, and you send numbers). Here is a [link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422) with details.

Reply:
I added the 422 code here because I was getting this error when the program tried to delete a file that didn't exist. Since the `if` block only checked for a 404 error, the program continued executing when it should have terminated.

<img width="870" alt="Screenshot 2023-12-31 at 12 02 14 PM" src="https://github.com/BigSamu/OpenSearch-bot/assets/109883576/a03f4017-8ef0-42c3-904a-38c036bb5f44">

Reply:
Here, it says we are passing a sha in the wrong format. When doing the refactoring of the deleteFileByPath() function, this would be resolved.

Reply:
This scenario will be addressed in the refactor of the `deleteFileByPath()` function.